hydra/core/plugins.py

```diff
@@ -160,6 +160,10 @@ def _scan_all_plugins(
                 path=mdl.__path__, prefix=mdl.__name__ + ".", onerror=lambda x: None
             ):
                 try:
+                    file_name = modname.rsplit(".", 1)[-1]
+                    # If filename starts with "_", do not load the file.
+                    if file_name.startswith("_"):
+                        continue
```

Comment:
this is a module, not a file.
fix variable names and comment.

Reply:
Ahh yes :)